src/assets/constants/components/Tile.jsx

```diff
@@ -0,0 +1,11 @@
+const Tile = ({ title, content, imageSrc, altText }) => {
+    return (
+        <section className="footer">
+            <h2>{title}</h2>
+            <p>{content}</p>
```

Comment:
In het voorbeeld heeft de eerste tile twee tekstblokken. Met een children property had je dit kunnen nabootsen.